data/language/en-GB.txt

```diff
@@ -3724,6 +3724,8 @@ STR_6273    :Music
 STR_6274    :Can't set colour scheme...
 STR_6275    :{WINDOW_COLOUR_2}Station style:
 STR_6276    :{RED}{STRINGID} has guests getting stuck, possibly due to invalid ride type or operating mode.
+STR_6277    :Multicore
```

Comment:
There is nothing "multicore" in here, you're simply spawning more threads which OS might then map to physical threads. `s/core/thread/g`

Reply:
Its a common term used in games actually, probably because most people wouldn't know the difference. What would you propose instead?

Reply:
`s/core/thread/`

Reply:
Yes. Make sure to change it in all the places.